packages/scandipwa/src/route/CartPage/CartPage.config.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+// eslint-disable-next-line import/prefer-default-export
+export const CART_URL = '/cart';
```

Comment:
Now, when you have created the constant for the cart URL, can you please replace all `'/cart'` occurrences in the code with this constant? :)

Reply:
I think this is my easter eggs search XD XD